examples/play.rs

```diff
@@ -13,7 +13,10 @@ fn main() {
         options.gain = 0.5;
         graph.create_node(AudioNodeType::GainNode(options));
         graph.resume_processing();
-        thread::sleep(time::Duration::from_millis(5000));
+        thread::sleep(time::Duration::from_millis(2000));
+        graph.message_node(0, AudioNodeMessage::SetFloatParam(220.));
```

Comment:
How would the API look like for two-way messaging (e.g. a getter)?

Reply:
We probably send it a channel to respond through.